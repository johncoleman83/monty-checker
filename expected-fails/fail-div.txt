L7: division by zero
0
4
3
2
1
