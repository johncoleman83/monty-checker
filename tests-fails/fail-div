#!/bin/bash
#
# check fails division

montyfile="./bytecode-fails/fail-div.m"
correct="./expected-fails/fail-div.txt"

# send commands
./monty $montyfile > $ERROROUTPUTFILE 2> $OUTPUTFILE &
cat $correct > $EXPECTED

# wait a little bit
$SLEEP $SLEEPSECONDS

# check the result
check_diff

#error
#print_error

# clean up
stop_monty
