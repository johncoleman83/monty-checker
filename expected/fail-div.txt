L7: division by zero
